client/components/TestRun/index.jsx

```diff
@@ -295,41 +324,54 @@ class TestRun extends Component {
                             >
                                 Go to next test
                             </Button>
-                        </React.Fragment>
+                        </Fragment>
                     );
                 }
             } else {
-                content = (
-                    <React.Fragment>
-                        <div>Tests are complete.</div>
-                    </React.Fragment>
-                );
+                content = <div>Tests are complete.</div>;
             }
         } else {
             heading = (
-                <React.Fragment>
+                <Fragment>
                     <h2>{`${apg_example_name}`}</h2>
                     <h3>{`${at_name} ${at_version} with ${browser_name} ${browser_version}`}</h3>
-                </React.Fragment>
-            );
-            content = (
-                <React.Fragment>
-                    <div>No tests for this browser / AT combination</div>
-                </React.Fragment>
+                </Fragment>
             );
+            content = <div>No tests for this browser / AT combination</div>;
         }
 
         let modals = this.renderModals();
 
         return (
-            <React.Fragment>
-                <Helmet>
-                    <title>{`Testing ${apg_example_name} for ${at_name} ${at_version} with ${browser_name} ${browser_version} | ARIA-AT`}</title>
-                </Helmet>
-                {heading}
-                {content}
+            <Fragment>
+                <Container fluid>
+                    <Row>
+                        <Col>
+                            <Helmet>
```

Comment:
I think the Helmet doesn't have to be in a row and col, it's just for changing the title element of the webpage.